pkg/apis/v1alpha1/common_validations.go

```diff
@@ -60,10 +63,20 @@ func validateFieldSelectorRequirements(reqs []FieldSelectorRequirement, validPat
 		if !validPath(req.Key, validPaths, validPrefixes) {
 			return fmt.Errorf("requirement key [%s] is not a valid path", req.Key)
 		}
+
+		if err := validJsonpath(req.Key); err != nil {
```

Comment:
pretty nice - I wonder if this is something we should do for the `Cluster*Template` outputs (like `imagePath`, etc) given that one could end up messing up their jsonpaths there, thus, having some early validation could be helpful

wdyt?

Reply:
write up an issue!